gto/cli.py

```diff
@@ -472,6 +479,7 @@ def annotate(
     ),
     label: List[str] = Option(None, "--label", help="Labels to add to artifact"),
     description: str = Option("", "-d", "--description", help="Artifact description"),
+    auto_commit: bool = option_auto_commit,
```

Comment:
WDYT about using just `commit` instead of `auto_commit`? For CLI and API. And `push` instead of `auto_push`. Looks like it's sufficient and shorter.

Reply:
Very good idea!

However, may I suggest for this PR to stick to `auto_commit`, for consistency with the pre-existing `auto_push`?
After this PR is merged, I will create a separate PR to rename all of them :)

I don't like mixed up PRs, as you have may noticed :P